addons/material-forum-editor-buttons/icons.css

```diff
@@ -0,0 +1,163 @@
+}
+.markItUp .markItUpButton3 a {
+  background-image: url("https://fonts.gstatic.com/s/i/materialicons/format_underlined/v5/24px.svg?download=true") !important;
+  background-size: 20px;
+}
+
+.markItUp .markItUpButton4 a {
+  background-image: url("https://fonts.gstatic.com/s/i/materialicons/strikethrough_s/v6/24px.svg?download=true") !important;
+  background-size: 20px;
+}
+
+.markItUp .markItUpButton5 a {
+  background-image: url("https://fonts.gstatic.com/s/i/materialicons/insert_photo/v4/24px.svg?download=true") !important;
+  background-size: 20px;
+}
+.markItUp .markItUpButton6 a {
+  background-image: url("https://fonts.gstatic.com/s/i/materialicons/insert_link/v4/24px.svg?download=true") !important;
+  background-size: 20px;
+}
+
+.markItUp .markItUpButton7 a {
+  background-image: url("https://fonts.gstatic.com/s/i/materialicons/format_size/v4/24px.svg?download=true") !important;
+  background-size: 20px;
+}
+.markItUp .markItUpButton8 a {
+  background-image: url("https://fonts.gstatic.com/s/i/materialicons/format_list_bulleted/v4/24px.svg?download=true") !important;
+  background-size: 20px;
+}
+.markItUp .markItUpButton9 a {
+  background-image: url("https://fonts.gstatic.com/s/i/materialicons/format_list_numbered/v4/24px.svg?download=true") !important;
+  background-size: 20px;
+}
+.markItUp .markItUpButton10 a {
+  background-image: url("https://gist.githubusercontent.com/easrng/4f63ce27f1970632528143430698db89/raw/f20856e9c4ab9ea176207f4980556c3fa4ec87b4/listitem.svg") !important;
+  background-size: 20px;
+}
+
+.markItUp .markItUpButton11 a {
+  background-image: url("https://fonts.gstatic.com/s/i/materialicons/format_quote/v4/24px.svg?download=true") !important;
+  background-size: 20px;
+}
+.markItUp .markItUpButton12 > a {
+  background-image: url("https://fonts.gstatic.com/s/i/materialicons/insert_emoticon/v4/24px.svg?download=true") !important;
+  background-size: 20px;
+}
+.markItUp .scratchblocks-button > a {
+  background-image: url("https://raw.githubusercontent.com/Secret-chest/images/3fce8c34173468fe3e423e8a8e5a80cffbfab8f6/scratchblocks.svg") !important;
+  background-size: 20px;
+}
+.markItUp .browser-os-button a {
+  background-image: url("https://fonts.gstatic.com/s/i/materialicons/computer/v4/24px.svg?download=true") !important;
+  background-size: 20px;
+}
+.markItUp .clean a {
+  background-image: url("https://fonts.gstatic.com/s/i/materialicons/code/v4/24px.svg?download=true") !important;
+  background-size: 20px;
+}
+.markItUp .preview a {
+  background-image: url("https://fonts.gstatic.com/s/i/materialicons/done/v4/24px.svg?download=true") !important;
+  background-size: 20px;
+}
+#ibbBtn,
+#uploadButton {
+  background-image: url("https://fonts.gstatic.com/s/i/materialicons/cloud_upload/v4/24px.svg?download=true") !important;
+  background-size: 20px;
+}
+.markItUpHeader ul .markItUpDropMenu:before {
+  content: url("https://fonts.gstatic.com/s/i/materialicons/expand_more/v4/24px.svg?download=true");
```

Comment:
Would it be better to keep these images uploaded in scratch addons for load times?

Reply:
Just inline them as data uris

Reply:
@TheColaber Ok, I will upload the icons to SA.

Reply:
@secret-chest please use a data URL or include this inside your addon's folder

Reply:
@WorldLanguages I will do it tomorrow

Reply:
> @Secret-chest please use a data URL or include this inside your addon's folder

@WorldLanguages What is a data URL?

Reply:
See https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs

Reply:
@secret-chest ?

Reply:
@WorldLanguages Yes, I will do them today.